config/src/test/java/org/axonframework/config/EventProcessingModuleTest.java

```diff
@@ -207,18 +205,6 @@ void processorsDefaultToSubscribingWhenUsingSimpleEventBus() {
                                    .orElse(false));
     }
 
-    @Test
-    void assigningATrackingProcessorFailsWhenUsingSimpleEventBus() {
```

Comment:
This test would still apply for the PSEP. It just needs to be rewritten.

BTW, the reason this test is there, is to validate that people do not combine a Streaming Event Processor without having a default `StreamableEventSource`.